xbmc/addons/kodi-addon-dev-kit/include/kodi/api2/.internal/AddonLib_internal.hpp

```diff
@@ -0,0 +1,112 @@
+#pragma once
```

Comment:
Is a header file :) you mean the `.internal` yes can be changed to normal one. Was to prevent not for add-on development needed files to view, but right to use not hidden is better (was also not so easy on Mac to add it in xcode ;) )